pkgs/by-name/ns/nstool/package.nix

```
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nstool";
+  version = "1.9.0";
+
+  src = fetchFromGitHub {
+    owner = "jakcron";
+    repo = pname;
```

https://github.com/NixOS/nixpkgs/issues/277994
```suggestion
    repo = "nstool";
```